app/schemas/ogc-api-coverages/1.0.0/ogc-api-coverages-v1.0.0.yml

```diff
@@ -911,3 +913,14 @@ components:
       required: false
       schema:
         type: string
+    granuleName:
+      name: granuleName
+      in: query
+      description: |
+        passed to the CMR search as the readable_granule_name parameter. Supports * and ? wildcards
+        for multiple and single character matches. Wildcards can be used any place in the name,
+        but leading wildcards are discouraged as they require a lot of resources for the underlying
+        search.
+      required: false
+      schema:
+        type: string
```

Comment:
Do we only want to allow a single `granuleName` value for this parameter? CMR `readable_granule_name` supports multiple values in an array.

Reply:
It supports a comma separated list

Reply:
I think the code actually support multiple values in an array. So we need to update here to reflect that.

Reply:
as well as multiple entries

Reply:
I see. This is not obvious without reading the actual parsing function in the code. As an API doc, we want to make this clear here. It is also not following the pattern set by the `granuleId` parameter. If there are people actually using the comma separated string for `granuleId`, we probably should update the documentation for `granuleId` as well.

Reply:
This uses exactly the pattern that granuleId uses.

Reply:
`granuleId` is of type array:
```
granuleId:
      name: granuleId
      in: query
      description: The CMR Granule ID for the granule which should be retrieved
      required: false
      style: form
      explode: true
      schema:
        type: array
        items:
          type: string
          minLength: 1
```

Reply:
Agree with Leo that the schema should be the same as granuleId - specifically:
```
      style: form
      explode: true
      schema:
        type: array
        items:
          type: string
          minLength: 1
```
instead of
```
     schema:
        type: string
```

Reply:
I see - my mistake. The code that handles the multi-value parameters is the same, but the schema was not. Fixed.